products/cakephp.md

```diff
@@ -260,14 +302,10 @@ the last major release cycle have security support.
 
 ## PHP Support
 
-| Version    | Min PHP | Max PHP |
-|------------|---------|---------|
-| 5.0        | 8.1     | Latest  |
-| 4.4 - 4.5  | 7.4     | Latest  |
-| 4.0 - 4.3  | 7.2     |         |
-| 3.4 - 3.10 | 5.6     | 7.4     |
-| 3.2 - 3.3  | 5.5     |         |
-| 3.0 - 3.1  | 5.4     |         |
-| 2          | 5.4     | 7.4     |
+{% include table.html
+labels="Release,Supported PHP versions"
+fields="releaseCycle,supportedPhpVersions"
+types="string,string"
+rows=page.releases %}
```

Comment:
```suggestion
{%- assign collapsedCycles = page.releases | collapse_cycles:"supportedPhpVersions"," - " %}
{% include table.html
labels="Release,Supported PHP versions"
fields="releaseCycle,supportedPhpVersions"
types="string,string"
rows=collapsedCycles %}
```

This would allow to merge similar lines together and to reduce the table size.

Reply:
Good hint, thx